source/view.ts

```diff
@@ -753,7 +753,14 @@ class View extends Backbone.View<Backbone.Model> {
 
         });
 
-        $container.css('display', displayCss);
+        if (!displayCss) {
+            $container.each((index, elem: HTMLElement) => {
+                elem.style.display = ''; // Fix because jQuery doesn't remove inline style display: none if displayMode is null or undefined.
+            });
+        } else {
+            $container.css('display', displayCss);
+        }
+
```

Comment:
```
$container.css('display', displayCss || '');
```
Could this work instead? Might be more efficient.
See [this SO answer](https://stackoverflow.com/a/4036868).